modules/swagger-codegen/src/main/resources/python/model.mustache

```diff
@@ -89,9 +89,10 @@ class {{classname}}(object):
 {{/isEnum}}
 {{^isEnum}}
 {{#hasValidation}}
-
+{{#required}}
         if not {{name}}:
```

Comment:
This particular issue was blocking me, since I got the invalid value due to a 0.